### ISSUE TITLE
All alert columns should be selected in distribution-rules.xml

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-broker#264]
**Describe the issue**
We currently distribute only a subset of the original alert columns, plus added columns. But we should distribute all available alert information (after the filtering step).

Action items: 
- Select all columns by default in `conf/distribution-rules.xml` (Actually maybe just removing the `select` section if it is simpler). 
- Make sure we send back the alerts with minimal schema change with respect to the incoming alerts, that is: the original alert schema plus the added column in the top level.
- Propagate all these change in `fink-client` as well.

**Comment by cAbhi15** [comment]
@JulienPeloton by all, do you mean including the cutout stamps?
It shouldn't be very difficult to include all the columns (even with `select`) as selecting the parent column would do the job:
```
<select>
    <column name="objectId"/>
    <column name="candid"/>
    <column name="candidate"/>
    <column name="prv_candidates"/>
    <column name="cross_match_alerts_per_batch"/>
</select>
```

**Comment by JulienPeloton** [comment]
@cAbhi15 yes ideally the timestamp should be included (this is one of the properties users are used to inspect a lot).

**Comment by JulienPeloton** [comment]
Fix in #269
